torch/fx/experimental/symbolic_shapes.py

```diff
@@ -312,6 +312,39 @@ def _constrain_symbol_range(shape_env, s: sympy.Symbol, min: int, max: int):
     else:
         shape_env.var_to_range[s] = ValueRanges(min, max)
 
+    shape_env.runtime_var_to_range[s] = shape_env.var_to_range[s]
+
+def _constrain_unbacked_symint_for_size(a):
+    if isinstance(a, int):
+        return
```

Comment:
This is weirdly asymmetric with `_constrain_symbol_range`. Don't you want to assume you're getting a sympy.Symbol as argument?

Reply:
(and then the enclosing code needs to do a >= 0 test if you do get an int)